### PR TITLE
Brings back fleet jackets, polos, et cetera

### PR DESCRIPTION
**Pull request by Drew156**
## About the pull request
Puts the Fleet Jacket and Polo and Pants back into the uniform vendor under utility extra

## Why it's good for the game
Brings back the uniform variety that the last branch didn't really replace when it made these admin spawn only.  Also looks cool.

## Did you test it?
Yes https://i.gyazo.com/dd943613aeb66729ee211b5ceba3a36e.png

## Changelog

:cl:
rscadd: Readds Alternative Fleet Utility Uniform Items to the uniform vendor
/:cl:

### CODE REVIEW

**Comment by Cupax3** [comment]
Heyo, please follow the PR template.

File: maps/torch/datums/uniforms_fleet.dm [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
 	utility_under = /obj/item/clothing/under/solgov/utility/fleet/command
-	utility_extra = list(
-				/obj/item/clothing/under/solgov/utility/fleet/combat/command,
-				/obj/item/clothing/head/beret/solgov/fleet/command,
-				/obj/item/clothing/head/ushanka/solgov/fleet,
-				/obj/item/clothing/suit/storage/hooded/wintercoat/solgov/fleet,
-				/obj/item/clothing/head/soft/solgov/fleet,
-				/obj/item/clothing/gloves/thick/duty/solgov/cmd
-			)
+	utility_extra = list(/obj/item/clothing/under/solgov/utility/fleet/officer/command,
```
Comment (Cupax3) [inline maps/torch/datums/uniforms_fleet.dm]: This entire section of indentation is incorrect and one too far, not to mention you have stuff on the same line as the opening and closing parenthesis. Please fix.

Comment (Cupax3) [reply maps/torch/datums/uniforms_fleet.dm]: Also, this applies to all modified lists.

**Review by Cupax3 — CHANGES_REQUESTED** [review]
Please adjust your changed files to be in the correct format, thank you.

File: maps/torch/datums/uniforms_fleet.dm [hunk 2]
```diff
@@ -309,15 +307,15 @@
...
-	utility_extra = list(
-				/obj/item/clothing/under/solgov/utility/fleet/combat/supply,
-				/obj/item/clothing/head/beret/solgov/fleet/command,
-				/obj/item/clothing/head/beret/solgov/fleet/supply,
-				/obj/item/clothing/head/ushanka/solgov/fleet,
-				/obj/item/clothing/suit/storage/hooded/wintercoat/solgov/fleet,
-				/obj/item/clothing/head/soft/solgov/fleet,
-				/obj/item/clothing/gloves/thick/duty/solgov/sup
-			)
+		utility_extra = list(/obj/item/clothing/under/solgov/utility/fleet/officer/command,
```
Comment (Shadowtail117) [inline maps/torch/datums/uniforms_fleet.dm]: This is where the Dreamchecker checks fail (line 308). Inconsistent indentation leads to a compilation error.

**Review by Shadowtail117 — CHANGES_REQUESTED** [review]

**Comment by Cupax3** [comment]
This will be marked as Stale in 1 day.

**Comment by Cupax3** [comment]
@OriginalNaym Heyho, your PR is now marked as **Stale**. If you do not reply within 14 days, it will be closed. You are free to reopen it later.

**Comment by Drew156** [comment]
The code should be fixed now, at least it ran properly on a local server.  I'm gonna wait to see if the checks work and then change the formatting

**Comment by Drew156** [comment]
Formatting's fixed I think we good

File: maps/torch/datums/uniforms_fleet.dm [hunk 3]
```diff
@@ -3,24 +3,20 @@
...
-				/obj/item/clothing/head/soft/solgov/fleet,
-				/obj/item/clothing/gloves/thick/duty/solgov/cmd
-			)
+	utility_extra = list(/obj/item/clothing/head/beret/solgov/fleet/command,
+						 /obj/item/clothing/head/ushanka/solgov/fleet,
+						 /obj/item/clothing/suit/storage/hooded/wintercoat/solgov/fleet,
+						 /obj/item/clothing/head/soft/solgov/fleet,
+						 /obj/item/clothing/gloves/thick/duty/solgov/cmd,
+						 /obj/item/clothing/under/solgov/utility/fleet/polopants/command,
+						 /obj/item/clothing/suit/storage/jacket/solgov/fleet/command)
```
Comment (Ilysen) [inline maps/torch/datums/uniforms_fleet.dm]: Is there a reason for the uh, super intense indentation? This seems to be something occurring throughout the file. If there's no reason for it, I'd request you tone it down a little and being it more in line with other formatting.

**Comment by Ilysen** [comment]
The code *looks* workable, but the indentation's a little crazy, like I mentioned. I can't speak on the content because I play almost exclusively contractor and I don't know how dress uniforms are, so you'd best have someone else comment on that front.

**Comment by Drew156** [comment]
Just fixed the indentation back to what it was before.  Honestly, I think it'd be possible just to put the whole list on one line like dress extras do, but I'm not gonna at least for now because I'm not all that confident with what I'm doing yet and I know the list as it stands works.

Content wise: This stuff might have not been exactly what it was modeled after when Bay first introduced these parts, but there's some uniforms irl that are sort of like these

The jacket: similar to [this](https://i.gyazo.com/a46f3f95201534f4ea867d5dec53b60a.png) fleece liner the US Army uses
Polo and pants: similar to [this](https://upload.wikimedia.org/wikipedia/commons/8/8c/Web_060714-N-8629M-163_Training.jpg) version of the naval combat uniform Canada uses

**Comment by Cupax3** [comment]
> Just fixed the indentation back to what it was before. Honestly, I think it'd be possible just to put the whole list on one line like dress extras do, but I'm not gonna at least for now because I'm not all that confident with what I'm doing yet and I know the list as it stands works.
> 
> Content wise: This stuff might have not been exactly what it was modeled after when Bay first introduced these parts, but there's some uniforms irl that are sort of like these
> 
> The jacket: similar to [this](https://i.gyazo.com/a46f3f95201534f4ea867d5dec53b60a.png) fleece liner the US Army uses
> Polo and pants: similar to [this](https://upload.wikimedia.org/wikipedia/commons/8/8c/Web_060714-N-8629M-163_Training.jpg) version of the naval combat uniform Canada uses

Gonna mention it here, but please direct your attention to Discord when able - your changes have some readability and convention issues. This is DNM until addressed.

File: maps/torch/datums/uniforms_fleet.dm [hunk 4]
```diff
@@ -4,23 +4,20 @@
...
-				/obj/item/clothing/head/soft/solgov/fleet,
-				/obj/item/clothing/gloves/thick/duty/solgov/cmd
-			)
+				 /obj/item/clothing/head/beret/solgov/fleet/command,
+				 /obj/item/clothing/head/ushanka/solgov/fleet,
+				 /obj/item/clothing/suit/storage/hooded/wintercoat/solgov/fleet,
+				 /obj/item/clothing/head/soft/solgov/fleet,
+				 /obj/item/clothing/gloves/thick/duty/solgov/cmd,
+				 /obj/item/clothing/under/solgov/utility/fleet/polopants/command,
+				 /obj/item/clothing/suit/storage/jacket/solgov/fleet/command)
```
Comment (Cupax3) [inline maps/torch/datums/uniforms_fleet.dm:13]: Please put the closing bracket on a new line.

File: maps/torch/datums/uniforms_fleet.dm [hunk 5]
```diff
@@ -4,23 +4,20 @@
...
 	service_hat = /obj/item/clothing/head/solgov/dress/fleet/command
 	service_over = /obj/item/clothing/suit/storage/solgov/service/fleet/officer
 
 	dress_over = /obj/item/clothing/suit/storage/solgov/dress/fleet/officer
 	dress_hat = /obj/item/clothing/head/solgov/dress/fleet/command
-	dress_extra = list(
-		/obj/item/material/sword/replica/officersword,
-		/obj/item/clothing/head/beret/solgov/fleet/dress/command
-	)
+	dress_extra = list(/obj/item/material/sword/replica/officersword, /obj/item/clothing/head/beret/solgov/fleet/dress/command)
```
Comment (Cupax3) [inline maps/torch/datums/uniforms_fleet.dm:20]: Please revert this.

File: maps/torch/datums/uniforms_fleet.dm [hunk 6]
```diff
@@ -42,13 +39,13 @@
...
-				/obj/item/clothing/head/soft/solgov/fleet,
-				/obj/item/clothing/gloves/thick/duty/solgov/eng
-			)
+				 /obj/item/clothing/head/beret/solgov/fleet/engineering,
+				 /obj/item/clothing/head/ushanka/solgov/fleet,
+				 /obj/item/clothing/suit/storage/hooded/wintercoat/solgov/fleet,
+				 /obj/item/clothing/head/soft/solgov/fleet,
+				 /obj/item/clothing/gloves/thick/duty/solgov/eng,
+				 /obj/item/clothing/under/solgov/utility/fleet/polopants,
+				 /obj/item/clothing/suit/storage/jacket/solgov/fleet)
```
Comment (Cupax3) [inline maps/torch/datums/uniforms_fleet.dm:48]: Same as for +7 to +13

File: maps/torch/datums/uniforms_fleet.dm [hunk 7]
```diff
@@ -4,23 +4,20 @@
...
-				/obj/item/clothing/suit/storage/hooded/wintercoat/solgov/fleet,
-				/obj/item/clothing/head/soft/solgov/fleet,
-				/obj/item/clothing/gloves/thick/duty/solgov/cmd
-			)
+				 /obj/item/clothing/head/beret/solgov/fleet/command,
+				 /obj/item/clothing/head/ushanka/solgov/fleet,
+				 /obj/item/clothing/suit/storage/hooded/wintercoat/solgov/fleet,
+				 /obj/item/clothing/head/soft/solgov/fleet,
+				 /obj/item/clothing/gloves/thick/duty/solgov/cmd,
+				 /obj/item/clothing/under/solgov/utility/fleet/polopants/command,
```
Comment (Cupax3) [inline maps/torch/datums/uniforms_fleet.dm:12]: These leading spaces are incorrect, please fix.

File: maps/torch/datums/uniforms_fleet.dm [hunk 8]
```diff
@@ -67,35 +64,26 @@
 	service_over = /obj/item/clothing/suit/storage/solgov/service/fleet/snco
 
 	dress_over = /obj/item/clothing/suit/storage/solgov/dress/fleet/snco
-	dress_hat = /obj/item/clothing/head/solgov/dress/fleet
```
Comment (Cupax3) [inline maps/torch/datums/uniforms_fleet.dm:70]: Why was the NCO cap removed for engi NCO?

File: maps/torch/datums/uniforms_fleet.dm [hunk 9]
```diff
@@ -67,35 +64,26 @@
...
-	dress_extra = list(
-		/obj/item/material/sword/replica/officersword/pettyofficer,
-		/obj/item/clothing/head/beret/solgov/fleet/dress
-	)
+	dress_extra = list(/obj/item/material/sword/replica/officersword/pettyofficer, /obj/item/clothing/head/beret/solgov/fleet/dress)
```
Comment (Cupax3) [inline maps/torch/datums/uniforms_fleet.dm:67]: Same as for +20

File: maps/torch/datums/uniforms_fleet.dm [hunk 10]
```diff
@@ -67,35 +64,26 @@
...
-				/obj/item/clothing/gloves/thick/duty/solgov/eng
-			)
-
+				 /obj/item/clothing/head/beret/solgov/fleet/command,
+				 /obj/item/clothing/head/ushanka/solgov/fleet,
+				 /obj/item/clothing/suit/storage/hooded/wintercoat/solgov/fleet,
+				 /obj/item/clothing/head/soft/solgov/fleet,
+				 /obj/item/clothing/gloves/thick/duty/solgov/eng,
+				 /obj/item/clothing/under/solgov/utility/fleet/polopants,
+				 /obj/item/clothing/suit/storage/jacket/solgov/fleet)
```
Comment (Cupax3) [inline maps/torch/datums/uniforms_fleet.dm:80]: See other comments.

File: maps/torch/datums/uniforms_fleet.dm [hunk 11]
```diff
@@ -67,35 +64,26 @@
...
 	service_hat = /obj/item/clothing/head/solgov/dress/fleet/command
 	service_over = /obj/item/clothing/suit/storage/solgov/service/fleet/officer
 
 	dress_over = /obj/item/clothing/suit/storage/solgov/dress/fleet/officer
 	dress_hat = /obj/item/clothing/head/solgov/dress/fleet/command
-	dress_extra = list(
-		/obj/item/material/sword/replica/officersword,
-		/obj/item/clothing/head/beret/solgov/fleet/dress/command
-	)
+	dress_extra = list(/obj/item/material/sword/replica/officersword, /obj/item/clothing/head/beret/solgov/fleet/dress/command)
```
Comment (Cupax3) [inline maps/torch/datums/uniforms_fleet.dm:86]: See other comments.

File: maps/torch/datums/uniforms_fleet.dm [hunk 12]
```diff
@@ -119,16 +107,16 @@
 	name = "Fleet security"
 	departments = SEC
 
-	utility_under = /obj/item/clothing/under/solgov/utility/fleet/security
```
Comment (Cupax3) [inline maps/torch/datums/uniforms_fleet.dm:122]: Why was this moved in position to +119?

File: maps/torch/datums/uniforms_fleet.dm [hunk 13]
```diff
@@ -119,16 +107,16 @@
...
-			)
-
+				 /obj/item/clothing/head/beret/solgov/fleet/security,
+				 /obj/item/clothing/head/ushanka/solgov/fleet,
+				 /obj/item/clothing/suit/storage/hooded/wintercoat/solgov/fleet,
+				 /obj/item/clothing/head/soft/solgov/fleet,
+				 /obj/item/clothing/gloves/thick/duty/solgov/sec,
+				 /obj/item/clothing/under/solgov/utility/fleet/security,
+				 /obj/item/clothing/under/solgov/utility/fleet/polopants/security,
+				 /obj/item/clothing/suit/storage/jacket/solgov/fleet/security)
```
Comment (Cupax3) [inline maps/torch/datums/uniforms_fleet.dm:118]: See other comments.

File: maps/torch/datums/uniforms_fleet.dm [hunk 14]
```diff
@@ -138,10 +126,6 @@
 	service_over = /obj/item/clothing/suit/storage/solgov/service/fleet
 
 	dress_over = /obj/item/clothing/suit/storage/solgov/dress/fleet
-	dress_extra = list(
-		/obj/item/clothing/head/beret/solgov/fleet/dress
-	)
-
```
Comment (Cupax3) [inline maps/torch/datums/uniforms_fleet.dm:144]: Why was this removed?

File: maps/torch/datums/uniforms_fleet.dm [hunk 15]
```diff
@@ -151,26 +135,20 @@
 	service_over = /obj/item/clothing/suit/storage/solgov/service/fleet/snco
 
 	dress_over = /obj/item/clothing/suit/storage/solgov/dress/fleet/snco
-	dress_hat = /obj/item/clothing/head/solgov/dress/fleet
```
Comment (Cupax3) [inline maps/torch/datums/uniforms_fleet.dm:154]: Why was this removed?

File: maps/torch/datums/uniforms_fleet.dm [hunk 16]
```diff
@@ -151,26 +135,20 @@
...
-	dress_extra = list(
-		/obj/item/material/sword/replica/officersword/pettyofficer,
-		/obj/item/clothing/head/beret/solgov/fleet/dress
-	)
+	dress_extra = list(/obj/item/material/sword/replica/officersword/pettyofficer, /obj/item/clothing/head/beret/solgov/fleet/dress)
```
Comment (Cupax3) [inline maps/torch/datums/uniforms_fleet.dm:138]: See other comments.

File: maps/torch/datums/uniforms_fleet.dm [hunk 17]
```diff
@@ -151,26 +135,20 @@
...
-				/obj/item/clothing/gloves/thick/duty/solgov/sec
-			)
-
+				 /obj/item/clothing/head/beret/solgov/fleet/command,
+				 /obj/item/clothing/head/ushanka/solgov/fleet,
+				 /obj/item/clothing/suit/storage/hooded/wintercoat/solgov/fleet,
+				 /obj/item/clothing/head/soft/solgov/fleet,
+				 /obj/item/clothing/gloves/thick/duty/solgov/sec,
+				 /obj/item/clothing/under/solgov/utility/fleet/polopants/security,
+				 /obj/item/clothing/suit/storage/jacket/solgov/fleet/security)
```
Comment (Cupax3) [inline maps/torch/datums/uniforms_fleet.dm:151]: See other comments.

Comment (Cupax3) [reply maps/torch/datums/uniforms_fleet.dm:151]: This also removes a following empty line.

File: maps/torch/datums/uniforms_fleet.dm [hunk 18]
```diff
@@ -200,15 +178,16 @@
 	name = "Fleet medical"
 	departments = MED
 
-	utility_under = /obj/item/clothing/under/solgov/utility/fleet/medical
```
Comment (Cupax3) [inline maps/torch/datums/uniforms_fleet.dm:203]: Why was this moved?

**Review by Cupax3 — CHANGES_REQUESTED** [review]
This PR includes

1. Undocumented edits
2. Generally poor edits, although likely via automated systems
3. Removals which are not documented

I stopped commenting the cases mid-way through because at this point these have become comments on all changes made. Please address them all and resolve the comment conversations before requesting a review again.
If you need help, ping me on Discord.

**Comment by Drew156** [comment]
I think I'm gonna re-do this manually because I copied a lot of old baycode that included whatever another codebase needed to make it work, which is probably why so much stuff is out of place.

So this is closed, I'll try to have a new one up either today or tomorrow

**Pull request closed without merging**